lms/app.py

```diff
@@ -25,7 +25,12 @@ def create_app(global_config, **settings):  # pylint: disable=unused-argument
     config.add_tween(
         "pyramid_exclog.exclog_tween_factory", under="pyramid_tm.tm_tween_factory"
     )
-    config.add_settings({"exclog.extra_info": True})
+    config.add_settings(
+        {
+            "exclog.extra_info": True,
+            "exclog.ignore": ["lms.services.exceptions.OAuth2TokenError"],
+        }
+    )
```

Comment:
:+1: I think this makes sense -- `OAuth2TokenError`'s are a normal and expected (and frequent) part of the app's operation. We don't need a bunch of info logged about them for debugging purposes.

I believe this will affect both production and development the same, right? :+1:

Reply:
What about Sentry? I can't remember whether `OAuth2TokenError`'s are going there currently or are they already filtered out?

Reply:
I don't see them in sentry but I can't find how they are being filtered.